### PR TITLE
Miscellaneous updates specific to z/TPF

### PR DESCRIPTION
**Pull request by jjohnst-us**
z/TPF had a build path change which needs to be propagated to the
build (redhat subpath is now gnu).  Additionally, a sleep needed
to be added as part of the z/TPF specific ddrgen and codegen to
allow for some build services to complete.

Signed-off-by: James D Johnston <jjohnst@us.ibm.com>

### CODE REVIEW

**Comment by jjohnst-us** [comment]
Is there a way I can tell which files have the bad copyright?

**Comment by DanHeidinga** [comment]
> Is there a way I can tell which files have the bad copyright?

Normally, I'd say click the details link and then look at the console output in Jenkins but this build failed with an OOM so I'll relaunch

**Comment by DanHeidinga** [comment]
Jenkins line endings check

**Comment by DanHeidinga** [comment]
Jenkins copyright check

File: runtime/port/sysvipc/j9sharedhelper.c [hunk 1]
```diff
@@ -293,7 +293,11 @@ cleanSharedMemorySegments(struct J9PortLibrary* portLibrary)
 			return;
 		}
 	}
-#else
+#elif defined(J9ZTPF) /* defined(J9ZOS390) */
+		/* The z/TPF platform does not support the ipcs command */
```
Comment (DanHeidinga) [inline runtime/port/sysvipc/j9sharedhelper.c]: This looks to be indented one level too deep

**Comment by DanHeidinga** [comment]
Jenkins copyright check

**Comment by DanHeidinga** [comment]
Jenkins line endings check

**Comment by jjohnst-us** [comment]
@DanHeidinga With respects to the sleep... it is a can of worms, but keep in mind it would only be a pain point for z/TPF if it does break in the future.  One layer of this complexity is because both our ddrgen and constgen are run as services  on our z/TPF system (on-line) whereas our build is offline on a Linux system.  The build pauses until these steps complete.  As part of currently internal continuous integration we launch a java program which runs the original build script and we have re-directed the standard output and error to a file.  The file contents is actually used as part of the build, because we need to 'find' the link statements and modify them to create z/TPF shared objects (part of the original build script).  So added the sleep because there is a lag as the file is being written by the time we actually go to do the link.  (In other we go to 'find' the link for ddr and constgen, but it wasn't written out yet).  20 seconds is plenty of a delay for the file to be completely written out when the link statement goes to execute.

Hope that helps clarify.

File: runtime/port/sysvipc/j9sharedhelper.c (hunk 1 above)
Comment (jjohnst-us) [reply runtime/port/sysvipc/j9sharedhelper.c]: Update pushed.

**Comment by DanHeidinga** [comment]
Jenkins test sanity zlinux jdk8

**Pull request merged**